src/serverenvironment.cpp

```diff
@@ -279,30 +279,67 @@ void LBMManager::applyLBMs(ServerEnvironment *env, MapBlock *block, u32 stamp)
 
 void fillRadiusBlock(v3s16 p0, s16 r, std::set<v3s16> &list)
 {
+	const s16 r2 = r * r;
 	v3s16 p;
 	for(p.X=p0.X-r; p.X<=p0.X+r; p.X++)
-		for(p.Y=p0.Y-r; p.Y<=p0.Y+r; p.Y++)
-			for(p.Z=p0.Z-r; p.Z<=p0.Z+r; p.Z++)
-			{
-				// limit to a sphere
-				if (p.getDistanceFrom(p0) <= r) {
-					// Set in list
-					list.insert(p);
-				}
-			}
+	for(p.Y=p0.Y-r; p.Y<=p0.Y+r; p.Y++)
+	for(p.Z=p0.Z-r; p.Z<=p0.Z+r; p.Z++)
+	{
```

Comment:
wrong code style

Reply:
This is explicitly allow, no?

Reply:
The nested loop is fine. Wrong formatted lines (spaces, indents or pointers) should be corrected if they're touched in a change.

Reply:
Spaces after 'for', spaces around '=', '<=', '+' and '-', and opening brace on previous line.